chapters/03_queueing_model/05_numeric_results_and_timings/main.tex

```diff
@@ -0,0 +1,331 @@
+although the recursive approach seems to be more linear in nature than the
+closed form method. 
+
+
+\subsubsection{Markov chain - Simulation comparison of steady state probabilities}
+
+Another comparison that can be made is the comparison between the steady state
+probabilities calculated using the Markov chain and the simulation.
+The steady state probabilities, defined in
+Section~\ref{sec:steady_state_probabilities}, are an essential measure since
+they are necessary in the calculation of all performance measures using the
+Markov chain model.
+Although, there is no need to calculate the steady state probabilities for the
+simulation, it is interesting to see how the two approaches compare.
+Figures~\ref{fig:comparison_steady_state_probabilities_1}
+to~\ref{fig:comparison_steady_state_probabilities_5} show a comparison between
+the steady state probabilities between the Markov chain and the simulation
+for different values of \(\mu\).
+The value of \(\mu\) gradually increases from \(\mu = 0.03\) to \(\mu = 0.27\)
+and for each one three plots are generated; the steady state probabilities
+generated by the Markov chain, the steady state probabilities generated by the
+simulation and the difference between the two.
+The rest of the parameters have the following values:
+
+\begin{multicols}{2}
+    \begin{itemize}
+        \item \(\lambda_1\) = 0.3
+        \item \(\lambda_2\) = 0.3
+        \item \(C\) = 5
+        \item \(T\) = 5
+        \item \(N\) = 20
+        \item \(M\) = 20
+    \end{itemize}    
+\end{multicols}
+
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth, trim=200 60 170 60, clip]{chapters/03_queueing_model/img/numeric_results_and_timings/steady_state_probabilities/main_1.pdf}
+    \caption{\(\mu = 0.03\)}
+    \label{fig:comparison_steady_state_probabilities_1}
+\end{figure}
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth, trim=200 60 170 60, clip]{chapters/03_queueing_model/img/numeric_results_and_timings/steady_state_probabilities/main_3.pdf}
+    \caption{\(\mu = 0.09\)}
+    \label{fig:comparison_steady_state_probabilities_2}
+\end{figure}
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth, trim=200 60 170 60, clip]{chapters/03_queueing_model/img/numeric_results_and_timings/steady_state_probabilities/main_5.pdf}
+    \caption{\(\mu = 0.15\)}
+    \label{fig:comparison_steady_state_probabilities_3}
+\end{figure}
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth, trim=200 60 170 60, clip]{chapters/03_queueing_model/img/numeric_results_and_timings/steady_state_probabilities/main_7.pdf}
+    \caption{\(\mu = 0.21\)}
+    \label{fig:comparison_steady_state_probabilities_4}
+\end{figure}
+
+\begin{figure}[H]
+    \includegraphics[width=\textwidth, trim=200 60 170 60, clip]{chapters/03_queueing_model/img/numeric_results_and_timings/steady_state_probabilities/main_9.pdf}
+    \caption{\(\mu = 0.27\)}
+    \label{fig:comparison_steady_state_probabilities_5}
+\end{figure}
```

Comment:
I wonder if these plots could appear as a grid? Looks a bit strange with the layout.

Reply:
Is something like this better?

![image](https://user-images.githubusercontent.com/20097997/195833719-ad4dbad1-c4ac-42db-a87a-99f21a493e4c.png)

Reply:
Leave the gridlines in. Just plot them in a single line

Reply:
This is what it looks like now 👍 

![image](https://user-images.githubusercontent.com/20097997/196000546-e8fac8b5-bfb7-41c8-9b90-1d763842e998.png)